src/style/row.ts

```diff
@@ -1,39 +1,45 @@
 import { css } from '@linaria/core';
 
 export const row = css`
-  display: contents;
-  line-height: var(--rdg-row-height);
-  background-color: var(--rdg-background-color);
+  @layer rdg.Row {
+    display: contents;
+    line-height: var(--rdg-row-height);
+    background-color: var(--rdg-background-color);
 
-  &:hover {
-    background-color: var(--rdg-row-hover-background-color);
-  }
+    &:hover {
+      background-color: var(--rdg-row-hover-background-color);
+    }
 
-  &[aria-selected='true'] {
-    background-color: var(--rdg-row-selected-background-color);
+    &[aria-selected='true'] {
+      background-color: var(--rdg-row-selected-background-color);
 
-    &:hover {
-      background-color: var(--rdg-row-selected-hover-background-color);
+      &:hover {
+        background-color: var(--rdg-row-selected-hover-background-color);
+      }
     }
```

Comment:
If it's ever needed, we could split this into `rdg.Row` and `rdg.RowSelected` layers for example, so that we can order them like this: `rdg.Row` -> `rdg.HeaderRow`/`rdg.SummaryRow`/ -> `rdg.RowSelected`

Summary rows doesn't try to change the `background-color` so it's fine, but I wonder about header rows 🤔